crates/biome_cli/src/execute/process_file.rs

```diff
@@ -27,6 +27,8 @@ pub(crate) enum FileStatus {
     Unchanged,
     /// While handling the file, something happened
     Message(Message),
+    /// A match was found while searching a file
+    Search(Message),
```

Comment:
Maybe `Match` or `SearchResult` is a better name? It doesn't seem to be used generically for messages from the `search` command.

Reply:
`SearchResult` sounds better, thank you 🙏